json/src/main/java/com/basistech/rosette/dm/jackson/AnnotatedDataModelModule.java

```diff
@@ -22,6 +22,7 @@
 import com.basistech.rosette.dm.BaseNounPhrase;
 import com.basistech.rosette.dm.CategorizerResult;
 import com.basistech.rosette.dm.Concept;
+import com.basistech.rosette.dm.LayoutRegion;
```

Comment:
The imports of `LayoutRegion` are alphabetized as if it were still `DataTypeRegion`, and the same for its mixin.